.rubocop_todo.yml

```diff
@@ -64,17 +64,12 @@ RSpec/ExampleLength:
 RSpec/ExpectInHook:
   Enabled: false
 
-# Offense count: 70
-# Configuration parameters: CustomTransform, IgnoreMethods.
-RSpec/FilePath:
-  Enabled: false
-
 # Offense count: 222
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle.
 # SupportedStyles: implicit, each, example
-RSpec/HookArgument:
-  Enabled: false
+# RSpec/HookArgument:
```

Comment:
Did you mean to just comment this out, or should we remove it?

Reply:
No, I meant to delete them.